tools/postman/validation/functions/patchRequestShouldHaveBody.js

```diff
@@ -0,0 +1,13 @@
+export default (request) => {
+  if (!request.method || request.method !== "PATCH") {
+    return;
+  }
+  const body = request.body;
+  if (!body || !body.raw) {
```

Comment:
not sure if you can use `?.` in the version of js you're using, but if you can you can replace it by
```js
if (!body?.raw) {
```

Reply:
Yes you can, thank you